common/decisions/goblinmoney_decisions.txt

```diff
@@ -0,0 +1,79 @@
+﻿get_a_loan_from_the_goblins_decision = {
+	title = get_a_loan_from_the_goblins_title			# override title, default: "<key>"; supports dynamic descriptions like in events (first_valid, ...); scope: character
+	picture = "gfx/interface/illustrations/decisions/goblo.dds"
+	ai_goal = no
+	ai_check_interval = 0
+	selection_tooltip = goblin_money_tooltip
+	desc = goblin_money_decision_desc
+	confirm_text = goblin_money_decision_confirm
+
+	is_shown = {
+                      
+                      highest_held_title_tier > tier_barony
+		NOT = {
+                     highest_held_title_tier > tier_barony

```

Comment:
![image](https://user-images.githubusercontent.com/74790101/226219604-f1ab00d2-fab8-4ddd-97a7-89adc440bf33.png)

you are causing a contradiction, this is why it does not show up at all.

Reply:
Done,test again before release if its possible